.github/workflows/prWorkflow.yml

```diff
@@ -31,4 +31,5 @@ jobs:
     uses: modusbox/github-actions-node/.github/workflows/testIntJob.yml@v0.0.2
 
   test_func:
-    uses: modusbox/github-actions-node/.github/workflows/testFuncJob.yml@v0.0.2
+    uses: modusbox/github-actions-node/.github/workflows/testFuncJob.yml@v0.0.4
```

Comment:
All jobs should be updated to v0.0.4 (i.e. see above this change)

Lets keep them in sync please.

Reply:
Done